readme.adoc

```diff
@@ -218,3 +220,12 @@ $NEO4J_HOME/bin/neo4j restart
 ----
 
 // end::readme[]
+
+ifndef::env-docs[]
+== Feedback
```

Comment:
I actually had the feedback section higher up b/c people don't tend to read longer documents and the call-to-action for testing/feedback is quite important for them to see. ...